### ISSUE TITLE
Data connection while connected to Dride Wi-Fi

### ISSUE THREAD

**Issue by szath75** [dride/dride-app-issues#51]
Not sure this is an issue of the device.
When I am connected to the Dride Wi-Fi on 80% of times I am not able to retrieve data (on 3G-4G) from Internet (i.e. WhatsApp), but on 20% of times data still work.
What should it be related to? Of course I would like to be able to still access internet.

**Comment by BilliTheKid** [comment]
> Not sure this is an issue of the device.
> When I am connected to the Dride Wi-Fi on 80% of times I am not able to retrieve data (on 3G-4G) from Internet (i.e. WhatsApp), but on 20% of times data still work.
> What should it be related to? Of course I would like to be able to still access internet.

Some other backers did report this error. So when connected and you open say Google Maps there is no data to run it? Mine seems fine.

**Comment by szath75** [comment]
Exactly, as I reported it happens on around the 80% of times. But sometimes data connection works.
This is not an app related problem, because the app is closed. It is related to the device probably.

**Comment by BilliTheKid** [comment]
> Exactly, as I reported it happens on around the 80% of times. But sometimes data connection works.
> This is not an app related problem, because the app is closed. It is related to the device probably.

The app needs to be running in the background in order for it to connect successfully. Again I'm on Android so nothing I can say really.

**Comment by szath75** [comment]
It's true! If the app is opened data transfer is working. Pity that I need to open the app every time I need it, it should work always as a background process instead.

**Comment by szath75** [comment]
Is it possible let Dride app work as a background process in order to keep the data connection active (the top would be if it starts when the wifi connection is established)?
The cell phone automatically connect to the Dride Wifi when I enter the car and data connection drops down automatically if I don't open the app.

**Comment by saoron** [comment]
This is a great idea! We're looking into it.
